Makefile

```diff
@@ -156,7 +156,7 @@ endif
 ginkgoBuild: generate
 	make -C ./test/e2e ginkgo-build
 
-test: generate ginkgoBuild
```

Comment:
Not sure why `make generate` was invoking `make ginkgoBuild`. As you can see, `ginkgoBuild` is a target against the `test/e2e` make spec, but `test` explicitly skips `test/e2e`.